.pre-commit-config.yaml

```diff
@@ -44,9 +44,11 @@ repos:
         name: prettier
         entry: prettier --write --prose-wrap always --print-width 120
         types: ["markdown"]
+        # Excluding modules/eks/cluster/README.md because it chokes on the output
+        # of the `terraform-docs` command for that file.
         exclude: |
           (?x)^(
-              README.md |
+              README.md | modules/eks/cluster/README.md |
```

Comment:
We don't want to ignore any specific file here. Instead, wrap the section you want ignored in the README directly.

See Slack thread respond and this ARB
https://github.com/orgs/cloudposse-corp/discussions/233

Reply:
I've pushed a fix for this now